### ISSUE TITLE
Geolite2 download not available after December 30, 2019

### ISSUE THREAD

**Issue by wmtech-1** [revive-adserver/revive-adserver#1141]
Since MaxMind does not allow public downloads any more since Dec. 30, the new GeoIP2 plugin cannot update the Geolite2 files any longer and presumably also is no longer installable because the initial database download will fail.

We need a way to use our license key for a free download and update of GeoLite2-City database.

**Comment by erikgeurts** [comment]
For background information: https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/

**Comment by erikgeurts** [comment]
We've begun testing a future version of the plugin that is capable of storing a MaxMind license key, and then use that to download the file and future updates of it using that license key. This is what the settings screen looks like:
![image](https://user-images.githubusercontent.com/5102461/71825555-65d96980-309c-11ea-8b49-d780fecb6a58.png)

**Comment by erikgeurts** [comment]
The new functionality appears to work, but I'm not 100% certain about the update process during the midnight maintenance task.

I upgraded an existing test instance to the v5.0.3 beta RC1. After entering the license key, I disabled and then re-enabled the plugin. As expected, the GeoLite2-City.mmdb was downloaded, and the GeoLite2-City.tar.gz.md5 was created. This was on Monday January 6th.

As far as I remember, MaxMind always publish their weekly updates on Tuesday. But the midnight maintenance on Wednesday did not pull an updated file, and neither did the midnight maintenance on Thursday.
